FAIMS/FAIMS3
# feat: app theming

## JIRA Ticket

[BSS-260](https://jira.csiro.au/browse/BSS-260): App Theming Setup
[BSS-134](https://jira.csiro.au/browse/BSS-134): Survey Page: Overall page layout update
[BSS-217](https://jira.csiro.au/browse/BSS-217): Survey List Page: List item styling
[BSS-215](https://jira.csiro.au/browse/BSS-215): Survey List Page: Top bar styling

## Description

Set up the ability to support multiple themes and switch between them. An example theme "bubble" is also included. Various components in the notebook list page have been updated to use different settings between the themes.

## Proposed Changes

- added theme environment variable
- moved theme files into default folder and created a theme file to manage exports of theme specific code
- updated imports to use new theme exports
- added bubble theme files
- updated current components in the notebook list page to use theme specific settings
- created survey card component
- created heading component

## How to Test

- log in to the app (if not already)
- navigate to the home page
- update `app/.env` to include `VITE_THEME=bubble` and save the file
- verify that the theme has changed
- (optional) navigate to a notebook page and verify that the theme has changed there as well
- update `app/.env` to include `VITE_THEME=default` and save the file
- verify that the theme has changed back to the default theme

## Additional Information

Some features present in the default theme notebook list page have not yet been implemented in the bubble theme.

## Checklist
